Puppetfile.tracking

```diff
@@ -391,124 +417,144 @@ mod 'simp-simp_ipa',
 #   :git => 'https://github.com/simp/pupmod-simp-simp_kubernetes',
 #   :ref => 'master'
 
+#FIXME does not support Puppet 5 yet
 mod 'simp-simp_logstash',
   :git => 'https://github.com/simp/pupmod-simp-simp_logstash',
-  :tag => '5.0.1'
+  :branch => 'master'
 
 mod 'simp-simp_nfs',
   :git => 'https://github.com/simp/pupmod-simp-simp_nfs',
-  :ref => '0.0.5'
+  :branch => 'master'
 
 mod 'simp-simp_openldap',
   :git => 'https://github.com/simp/pupmod-simp-simp_openldap',
-  :tag => '6.2.1'
+  :branch => 'master'
 
 mod 'simp-simp_options',
   :git => 'https://github.com/simp/pupmod-simp-simp_options',
-  :ref => '1.2.0'
+  :branch => 'master'
+
+mod 'simp-simp_pki_service',
+  :git => 'https://github.com/simp/pupmod-simp-simp_pki_service',
+  :branch => 'master'
 
 mod 'simp-simp_rsyslog',
   :git => 'https://github.com/simp/pupmod-simp-simp_rsyslog',
-  :tag => '0.2.0'
+  :branch => 'master'
 
+#FIXME does not support Puppet 5 yet
 mod 'simp-simp_snmpd',
   :git => 'https://github.com/simp/pupmod-simp-simp_snmpd',
-  :tag => '0.0.3'
+  :branch => 'master'
 
 mod 'simp-site',
   :git => 'https://github.com/simp/pupmod-simp-site',
-  :tag => '2.0.4'
+  :branch => 'master'
 
 mod 'simp-ssh',
   :git => 'https://github.com/simp/pupmod-simp-ssh',
-  :ref => '6.4.3'
+  :branch => 'master'
 
 mod 'simp-sssd',
   :git => 'https://github.com/simp/pupmod-simp-sssd',
-  :tag => '6.1.2'
+  :branch => 'master'
 
 mod 'simp-stunnel',
   :git => 'https://github.com/simp/pupmod-simp-stunnel',
-  :tag => '6.3.2'
+  :branch => 'master'
 
 mod 'simp-sudo',
   :git => 'https://github.com/simp/pupmod-simp-sudo',
-  :ref => '5.0.6'
+  :branch => 'master'
 
 mod 'simp-sudosh',
   :git => 'https://github.com/simp/pupmod-simp-sudosh',
-  :tag => '6.0.1'
+  :branch => 'master'
 
 mod 'simp-svckill',
   :git => 'https://github.com/simp/pupmod-simp-svckill',
-  :tag => '3.2.6'
+  :branch => 'master'
 
 mod 'simp-swap',
   :git => 'https://github.com/simp/pupmod-simp-swap',
-  :tag => '0.1.1'
+  :branch => 'master'
 
 mod 'simp-tcpwrappers',
   :git => 'https://github.com/simp/pupmod-simp-tcpwrappers',
-  :tag => '6.0.3'
+  :branch => 'master'
 
 mod 'simp-tftpboot',
   :git => 'https://github.com/simp/pupmod-simp-tftpboot',
-  :ref => '6.1.0'
+  :branch => 'master'
 
-# SIMP has made changes to this module that don't exist upstream
-# yet. (PR pending). Since may revert to saz/puppet-timezone at a
-# later date, working off of simp-master.
-mod 'simp-timezone',
+# Revert to saz/puppet-timezone for Puppet 5
+# TODO Get RPM triggers to work so that can upgrade from
+#      simp-timezone to saz-timezone
+mod 'saz-timezone',
```

Comment:
Due to the need for RPM triggers, and the complexity of getting this working in the Oct time frame, I think that we should just update our fork until we can properly fix the module deployment strategy.